packages/lib/src/components/Card/Card.tsx

```diff
@@ -28,6 +28,8 @@ export class CardElement extends UIElement<CardElementProps> {
                 ...props.configuration,
                 socialSecurityNumberMode: props.configuration?.socialSecurityNumberMode ?? 'auto',
             },
+            brandsConfiguration: props.brandsConfiguration ?? props?.configuration?.brandsConfiguration ?? {},
+            icon: props.icon ?? props?.configuration?.icon,
```

Comment:
Small comments: I would drop the `?` after the `props` in both lines. Also do you see some benefit in using the nullish concealing operator instead of using a regular `||` on this case? because the later means less weight.

Reply:
The size argument makes sense, I'll update this part 👍